src/App.js

```diff
@@ -4,27 +4,61 @@ import React, { useState, useEffect } from 'react';
 import axios from 'axios';
 
 
+
 function App() {
   const [fighters, setFighters] = useState([]);
```

Comment:
You might need a state for storing which was the fighter selected at some point ?